flow/Platform.actor.cpp

```diff
@@ -1085,12 +1093,12 @@ void getDiskStatistics(std::string const& directory,
 			throw platform_error();
 		}
 
-		currentIOs = queue_len;
-		busyTicks = (u_int64_t)ms_per_transaction;
-		reads = total_transfers_read;
-		writes = total_transfers_write;
-		writeSectors = total_blocks_read;
-		readSectors = total_blocks_write;
+		currentIOs += queue_len;
+		IOMilliSecs += (u_int64_t)ms_per_transaction;
+		reads += total_transfers_read;
+		writes += total_transfers_write;
+		writeSectors += total_blocks_read;
+		readSectors += total_blocks_write;
```

Comment:
This totally changes what it was before. Can you add this change to the PR description as well?

Reply:
This is point 3 in description. I think what it did was wrong.